src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int EMPTY_RHYMER_INDICATOR = -1;
+	public static final int STACK_CAPACITY = 12;
+	public static final int MAX_INDEX = 11;

```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).